core/src/main/java/com/oath/oak/Chunk.java

```diff
@@ -529,9 +531,7 @@ Handle pointToValue(OpData opData) {
         int expectedHandleIdx = opData.handleIndex;
         int foundHandleIdx = getEntryField(opData.entryIndex, OFFSET_HANDLE_INDEX);
 
-        if (foundHandleIdx == expectedHandleIdx) {
-            return null; // someone helped
-        } else if (foundHandleIdx < 0) {
+        if (foundHandleIdx < 0) {
```

Comment:
I am not sure it is correct to remove the comparison and check that someone else helped so the found handle index value is as expected. The following checks like "else if (operation == Operation.PUT_IF_ABSENT)" assume the found handle index is different. Also it doesn't looks correct to recursively call for pointToValue() after all in this case...

Reply:
reverted